internal/test/conversion.go

```diff
@@ -479,13 +479,25 @@ func (r *TestResources) NewCryostatWithWsConnectionsSpecV1Beta1() *operatorv1bet
 }
 
 func (r *TestResources) NewCryostatWithCommandConfigV1Beta1() *operatorv1beta1.Cryostat {
+	coreSVC := r.NewCryostatService()
+	coreIng := r.newNetworkConfigurationV1Beta1(coreSVC.Name, coreSVC.Spec.Ports[0].Port)
+	coreIng.Annotations["custom"] = "annotation"
+	coreIng.Labels["custom"] = "label"
+
+	grafanaSVC := r.NewGrafanaService()
+	grafanaIng := r.newNetworkConfigurationV1Beta1(grafanaSVC.Name, grafanaSVC.Spec.Ports[0].Port)
+	grafanaIng.Annotations["grafana"] = "annotation"
+	grafanaIng.Labels["grafana"] = "label"
+
```

Comment:
Could I suggest calling `NewCryostatWithIngressV1Beta1` and then augmenting it with the command config before returning? It would be a bit less duplication.